Wheres-Waldo-Frontend/src/components/HomePage.jsx

```diff
@@ -0,0 +1,132 @@
+import { Link } from "react-router-dom";
+import waldoBeach from "../assets/waldoBeach.jpg";
+import waldoDowntown from "../assets/waldoDowntown.jpg";
+import waldoFactory from "../assets/waldoFactory.jpg";
+import styled from "styled-components";
+import styles from "./HomePage.module.css";
+
+function HomePage() {
+	return (
+		<>
+			<Header />
+			<Body />
+		</>
+	);
+}
+
+function Header() {
+	const title = "Where's Waldo! A photo tagging game";
+	return (
+		<>
+			<div className="title">
+				<h1 role="title">{title}</h1>
+			</div>
+		</>
+	);
+}
+
+function Body() {
+	const gameRules =
+		"This is a photo tagging app. I'm putting the rules here. Select your option below";
+
+	const startGame = async (imageId) => {
+		try {
+			const response = await fetch(
+				`http://localhost:3000/image/${imageId}/gameStart`,
+				{
+					method: "POST",
+					headers: {
+						"Content-Type": "application/json",
+					},
+				}
+			);
+
+			return response;
+		} catch (error) {
+			console.log(error, "this is error");
```

Comment:
It's usually good to show the user an error if something fails, like maybe render a basic div like "The game failed to start, refresh and try again" or something. You can simulate an error by disabling network in the browser dev tools, or running this function after killing your backend